container.go

```diff
@@ -469,7 +470,7 @@ func (c *ContainerRequest) BuildOptions() (types.ImageBuildOptions, error) {
 		}
 
 		if modifiedTag != tag {
-			Logger.Printf("✍🏼 Replacing image with %s. From: %s to %s\n", is.Description(), tag, modifiedTag)
+			log.Default().Printf("✍🏼 Replacing image with %s. From: %s to %s\n", is.Description(), tag, modifiedTag)
```

Comment:
suggestion: Expose a `Printf` in the `log` package so it can have cleaner consumers:
```suggestion
			log.Printf("✍🏼 Replacing image with %s. From: %s to %s\n", is.Description(), tag, modifiedTag)
```

More below

Reply:
Will do!